AgentDiscoveries-Frontend/app/sass/styles.scss

```diff
@@ -34,6 +34,19 @@ body {
             color: $main-color;
         }
     }
+    .navbar-right {
+        position: relative;
+    }
+    .nav-avatar {
+        width: 30px;
+        height: 30px;
+        object-fit: cover;
+        position: absolute;
+        border-radius: 50%;
+        top: 50%;
+        transform: translateY(-50%);

```

Comment:
This transform feels.. odd. It's like we can't quite decide what our top is, so then we adjust it?

I'd be inclined to stick with either %s or px - we could explicitly accept that the height is 60% of the nav bar, so we want to adjust by `top: 20%` ... but then we have width not obviously equal to height.

So maybe we should accept that this is reasonably hard coded & we're not worrying about responsive design - accept that width = height = 30px, and top = 10px, and we don't need to transform y ?

Reply:
(Discussed offline - more than happy to leave!)